ksqldb-engine/src/test/java/io/confluent/ksql/schema/ksql/inference/DefaultSchemaInjectorTest.java

```diff
@@ -475,8 +510,8 @@ public void shouldGetSchemasWithoutSchemaId() {
     final ConfiguredStatement<CreateTable> result = injector.inject(ctStatement);
 
     // Then:
-    assertThat(result.getStatementText(), containsString("KEY_SCHEMA_ID=18"));
-    assertThat(result.getStatementText(), containsString("VALUE_SCHEMA_ID=5"));
+    assertThat(result.getStatementText(), not(containsString("KEY_SCHEMA_ID=18")));
+    assertThat(result.getStatementText(), not(containsString("VALUE_SCHEMA_ID=5")));
```

Comment:
Given the changes from above, should we add `containsString("KEY_AVRO_SCHEMA_FULL_NAME='myrecord'")` and similar for value?

Reply:
This test actually shouldn't `containsString("KEY_AVRO_SCHEMA_FULL_NAME='myrecord'")` since we only change the schema name when schema id is explicitly provided which is covered for example on line 438.

Reply:
On the first PR, you said:
> `givenKeyAndValueInferenceSupported` doesn't specify schema id.

This method is use in this test, but also in `shouldInjectForCsStatement()` above, and in the above test, `KEY_AVRO_SCHEMA_FULL_NAME` and `VALUE_AVRO_SCHEMA_FULL_NAME` are both added.

What do I miss?

Reply:
![thumbup](https://media.giphy.com/media/dykJfX4dbM0Vy/giphy.gif)

good catch. Let me fix it.